src/components/ProgressBarContainer.ts

```diff
@@ -116,19 +116,19 @@ export default class ProgressBarContainer extends Component<ProgressBarContainer
         return defaultValue;
     }
 
-    private getBootstrapStyle(mxObject?: mendix.lib.MxObject): BootstrapStyle {
+    private getStyle(mxObject?: mendix.lib.MxObject): BarStyle {
```

Comment:
should be => `getBarStyle` als it will confuse with the style of the widget itself